src/config/config.js

```diff
@@ -175,8 +175,11 @@ export default {
    */
   knownNotations: ['preferred-email-encoding@pgp.com', 'pka-address@gnupg.org'],
   /**
+   * Whether to use the indutny/elliptic library for NIST and Brainpool curves that are not supported by the available native crypto library.
```

Comment:
And secp256k1. Alternatively, maybe we can just say
```suggestion
   * Whether to use the indutny/elliptic library for curves (other than Curve25519) that are not supported by the available native crypto API.
```